hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/MappingJdbcSqlConnectorTest.java

```diff
@@ -68,6 +69,66 @@ public void createMappingWithExternalTableName() throws Exception {
         );
     }
 
+    @Test
+    public void createMappingWithExternalSchemaAndTableName() throws Exception {
+        executeJdbc("CREATE SCHEMA schema1");
+        createTable("schema1." + tableName);
+
+        String mappingName = "mapping_" + randomName();
+        createMapping("schema1\".\"" + tableName, mappingName);
+
+        assertRowsAnyOrder("SHOW MAPPINGS",
+                newArrayList(new Row(mappingName))
+        );
+
+        String expectedMappingQuery = "CREATE OR REPLACE EXTERNAL MAPPING \"hazelcast\".\"public\"" +
+                ".\"" + mappingName + "\" EXTERNAL NAME \"schema1\"" +
```

Comment:
may fail on windows due to newlines. BTW, AssertJ can ignore kind of line endings, but assertRowsAnyOrder does not use it

Reply:
b7f92b6dda6